styleguide/_themes/derek/scss/components/_rug.scss

```diff
@@ -1,10 +1,15 @@
 .rug-phone-numbers{
   margin-top:6px;
+  padding-left:5px;
+  a{
```

Comment:
Should this `a` have a class so we can reference it directly, such as `.rug-phone-numbers-link`.